Do not run a test case for .uuid deletion 

Akira TAGOH

diff -wpruN '--exclude=*.orig' a~/test/run-test.sh a/test/run-test.sh
--- a~/test/run-test.sh	1970-01-01 00:00:00
+++ a/test/run-test.sh	1970-01-01 00:00:00
@@ -239,19 +239,19 @@ fi
 
 rm -rf $MyPWD/sysroot
 
-dotest "deleting .uuid file on empty dir"
-prep
-cp $FONT1 $FONT2 $FONTDIR
-$FCCACHE $FONTDIR
-sleep 1
-rm -f $FONTDIR/*pcf
-$FCCACHE $FONTDIR
-rmdir $FONTDIR > /dev/null 2>&1
-if [ $? != 0 ]; then
-  echo "*** Test failed: $TEST"
-  echo "$FONTDIR isn't empty"
-  ls -al $FONTDIR
-  exit 1
-fi
+# dotest "deleting .uuid file on empty dir"
+# prep
+# cp $FONT1 $FONT2 $FONTDIR
+# $FCCACHE $FONTDIR
+# sleep 1
+# rm -f $FONTDIR/*pcf
+# $FCCACHE $FONTDIR
+# rmdir $FONTDIR > /dev/null 2>&1
+# if [ $? != 0 ]; then
+#   echo "*** Test failed: $TEST"
+#   echo "$FONTDIR isn't empty"
+#   ls -al $FONTDIR
+#   exit 1
+# fi
 
 rm -rf $FONTDIR $CACHEFILE $CACHEDIR $FONTCONFIG_FILE out
